fixtures/kanban-fixture.html

```
@@ -0,0 +1,19 @@
+<!DOCTYPE html><html><body><script>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)